pkg/api/resource/status.go

```diff
@@ -109,6 +109,8 @@ const (
 	StatusAPIGroupParentFailed
 	StatusAPIGroupParentKilled
 	StatusAPIGroupUpdateSkipped
+
+	StatusDataOOM
```

Comment:
This should be up with the other data statuses

Reply:
Best practice with enums is appending to the end only, to not break previous states